solo/hmac_secret.py

```diff
@@ -60,7 +61,7 @@ def make_credential(
     credential = attestation_object.auth_data.credential_data
     credential_id = credential.credential_id
     if output:
-        print(credential_id.hex())
+        print(base64.b64encode(credential_id).decode('ascii'))
```

Comment:
This is a breaking change, which we should avoid.

Reply:
true enough. better Idea? maybe add a base64 flag/option? it's just that base64 is kinda the default of credentials ideas in most places I have seen them occur

Reply:
Yeah, you can add a `--format [hex|base64|ascii95|emoji|...]` optional argument with default the existing behaviour. Or you can open a separate issue to change said default.